### PR TITLE
Draft of tentative schedule for 2021 November Training

### PR DESCRIPTION
**Pull request by Jen-OMalley**
Draft of schedule to link in email advertisement and blog post

### CODE REVIEW

**Comment by Jen-OMalley** [comment]
I'd like to be able to include a link to this when we announce the next training, hopefully in the next couple of days. As of now, this should be pretty much identical to the June 2021 training.

File: virtual-workshop/SCHEDULE.md [hunk 1]
```diff
@@ -2,16 +2,48 @@
 title: Virtual Workshop Schedule
 nav_title: Schedule
 ---
+A tentative schedule for the November 2021 CCDL Virtual Training Workshop appears below. ⚠️ Not all links are working yet! ⚠️
 
+
+*Note: All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt)*
```
Comment (jashapiro) [inline virtual-workshop/SCHEDULE.md:8]: This is correct, but I had to check! DST ends right after this training!

**Review by jashapiro — APPROVED** [review]
LGTM! (though I had to check on time zones).

File: virtual-workshop/SCHEDULE.md (hunk 1 above)
Comment (Jen-OMalley) [reply virtual-workshop/SCHEDULE.md:8]: Great. Thank you!

**Pull request merged**